melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/X006FjernInstitusjon.java

```diff
@@ -0,0 +1,13 @@
+package no.nav.melosys.eessi.models.sed.nav;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import no.nav.melosys.eessi.integration.eux.rina_api.dto.Institusjon;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class X006FjernInstitusjon {
+
+    private Institusjon institusjon;
```

Comment:
Jeg tenker at vi kanskje heller kunne hatt et flagg her, som sier om vi er fjernet fra BUCen eller ikke?
Da slipper vi å dra inn masse EESSI-spesifikk logikk i melosys-api om hvilken institusjonID vi har, men heller la det ansvaret ligge her hos melosys-eessi.

Reply:
👍

Reply:
Eneste er konverteringen fra Seden ifølge template: https://stash.adeo.no/projects/EESSI2/repos/eux/browse/eux-acl/src/main/resources/sedtemplates/v42/administrative/X006_v4.2.json

Så er dette feltet     

"RemovedParticipant": {
      "Institution": {
        "institutionID": "$nav.sak.fjerninstitusjon.institusjon.id",
        "institutionName": "$nav.sak.fjerninstitusjon.institusjon.navn"
      }

Reply:
Det er riktig felt ja. 
`$nav.sak.fjerninstitusjon.institusjon.id` er nok det vi må sjekke på der, for å se om det er vi som er fjernet fra BUCen

Reply:
Yes, da legger jeg inn flagg i melosyseessimelding og ikke her.